### PR TITLE
added factorial calculator recursively

### PR DESCRIPTION
**Pull request by Croustys**

### CODE REVIEW

File: Recursive/FactorialCalculator.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-const factorialRecursion = (n) => n === 0 ? 1 : n * factorialRecursion(n - 1);
+const factorialRecursion = (n) => n === 0 ? 1 : n * factorialRecursion(n - 1)
```
Comment (ruppysuppy) [inline Recursive/FactorialCalculator.js:1]: Don't use arrow function, write it in the form of a normal function instead

File: Recursive/FactorialCalculator.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
-const factorialRecursion = (n) => n === 0 ? 1 : n * factorialRecursion(n - 1);
+const factorialRecursion = (n) => n === 0 ? 1 : n * factorialRecursion(n - 1)
 
-//num can be anything you'd like
-const num = 3;
-console.log(factorialRecursion(num));
+// num can be anything you'd like
+const num = 3
+console.log(factorialRecursion(num))
```
Comment (ruppysuppy) [inline Recursive/FactorialCalculator.js:5]: Use
```javascript
console.log(factorialRecursion(3))
```

Add a few tests

**Review by ruppysuppy — CHANGES_REQUESTED** [review]

**Comment by Croustys** [comment]
@ruppysuppy created new PR and fixed your enquiries.

File: Recursive/FactorialCalculator.js (hunk 1 above)
Comment (Croustys) [reply Recursive/FactorialCalculator.js:1]: Not sure what's the problem with arrow functions, but sure.

**Comment by ruppysuppy** [comment]
Duplicate PR #445

**Pull request closed without merging**